bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Group.cs

```diff
@@ -62,7 +62,7 @@ public unsafe string ToHex()
             var status = NativeInterface.ElementModP.ToHex(Handle, out IntPtr pointer);
             if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
             {
-                Console.WriteLine($"ToHex Error Status: {status}");
+                throw new ElectionGuardException($"ToHex Error Status: {status}");
```

Comment:
Looks solid. Although, I think long term we can hopefully overall for the system decide upon errors and indicate an overall setup on errors the percolate through the system. 

Note for ElectionGuard 2.0: 
This could be a really valuable discussion point. Writing out all the different types of errors that could occur in the electionguard system and having this as a requirement of the core.

Reply:
Yes, and I'd like to see more use of the various built-in exceptions like `ArgumentException`, `ArgumentOutOfRangeException`, `FormatException`, and so on.  This is more of an intrim solution.

Reply:
This is also why the new custom exception class can take an inner exception so that we can use the built-in types as well.